.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: release
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Which version are we releasing? (format vX.Y.Z)'
+        required: true
+
+permissions:
+  contents: write
+
+jobs:
+    release:
+        name: "release"
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout repo
+            uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+            with:
+              fetch-depth: 0  
+
+          - name: Configure github-actions[bot]
+            run: |
+              git config user.name 'github-actions[bot]'
+              git config user.email 'github-actions[bot]@users.noreply.github.com'
+          
+          - name: Create and push tag
+            run: |
+              git tag -a "${{ github.event.inputs.release_version }}" -m "${{ github.event.inputs.release_version }}"
```

Comment:
we need multiple tags. example: v1.2.0, v1.2, v1

Reply:
if we are creating a tag for something that's not latest (say v1.1.1), we don't want to update v1. this can be a follow up though

Reply:
examples:
release v1.2.0 -> update v1.2.0, v1.2 and v1
release v1.1.1 (after v1.2.0 is out) -> update v1.1, v1.1.1 (not v1 since that tracks v1.2.0 now)

let me know if i can clarify further

Reply:
Made changes to push multiple tags based on input version. Looking into how to add a check if the release is after the next version is already out (v1.1.1 after v1.2.0 is out)

Reply:
I tried to add a check to compare minor versions when we release v1.1.1 after v1.2.0, and use latest ("https://github.com/ashnamehrotra/copa-action/releases/latest") to get v1.2.0 to compare. Im not sure if this will work because after this release latest would incorrectly point to v1.1.1?